### PR TITLE
Update How it works

### PR DESCRIPTION
**Pull request by mgrigoriev8109**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).

#### 1.Describe the changes made:

Added a paragraph to the How It Works section. Since a lot of students seem to be voicing the same concern over forgetting material after having finished FCC, and not feeling ready for the Google Project, it seems appropriate to explain that memorization is not an expectation, and that programming is very different from the educational model most people are used to (school, studying, notes, tests, failure/passing).

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

File: foundations/introduction/how_this_course_will_work.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ This curriculum works by pulling together the best content from across the inter
 
 Most lessons will contain questions that you should be able to answer before moving on.  Some of the lessons will ask you to complete exercises. In addition, we provide several projects throughout the curriculum to help you grow your understanding by actually building things.
 
+Try not too think of The Odin Project, or programming, as a class in school. It's not material you learn all at once, take a test, and then pass or fail. You can think of it as a snowball. You, yourself, are a snowball. You're rolling down a hill full of snow, and the further you roll, the more snow will stick to you. Sure, snow will also fall off of you, especially if it's not too wet and sticky, and you'll forget things often. But the more you keep rolling the more will stick to you, and the bigger and smarter you get.
```
Comment (twalton83) [inline foundations/introduction/how_this_course_will_work.md:23]:
```suggestion
Try not to think of The Odin Project, or programming, as a class in school. It's not material you learn all at once, take a test, and then pass or fail. You can think of it as a snowball. You, yourself, are a snowball. You're rolling down a hill full of snow, and the further you roll, the more snow will stick to you. Sure, snow will also fall off of you, especially if it's not too wet and sticky, and you'll forget things often. But the more you keep rolling the more will stick to you, and the bigger and smarter you get.
```

Comment (twalton83) [reply foundations/introduction/how_this_course_will_work.md:23]: Love the metaphor, but can we bring it back to literal terms and emphasize that memorization is not a goal?

**Review by twalton83 — CHANGES_REQUESTED** [review]

File: foundations/introduction/how_this_course_will_work.md (hunk 1 above)
Comment (ZachBaird) [reply foundations/introduction/how_this_course_will_work.md]: I agree with @twalton83 tbh, we have a lot of students think they need to memorize everything before moving on, and addressing that this isn't true could be really helpful.

Comment (twalton83) [reply foundations/introduction/how_this_course_will_work.md]: My suggestion was dismissed, but you have a typo still. It should be “to” not “too”.

**Review by twalton83 — APPROVED** [review]

**Pull request merged**